script/1_0_2current_en_script/Arcueid/Day 1/01_00_ARC01_9B.txt

```diff
@@ -37,7 +37,7 @@ Having a girl around my own age look after me isn't to my taste right now.
 Hisui bows deeply.
 The stabbing pain in my chest is caused by my selfish guilt.
 <Page11>
-"All right. From the way Akiha puts it, she's not going to let me be alone, and I don't think she'll let me get rid of you either. So I'll accept your help but―――"
+"All right. From the way Akiha puts it, she's not going to let me be alone. Plus, I don't think making her dismiss you just to make allowance for me is a good thing. So I'll accept your help but―――"//「分かった。秋葉のあの言いぶりじゃ俺を一人にはしないだろうし、そもそも俺の気持ちの問題で君を解雇させるのは良くはないと思う。ありがたくお世話になるけど―――」
```

Comment:
replace 'plus' with 'besides'

I think we can get rid of 'making her dismiss you' and just say 'dismissing you'

Reply:
I'll replace the plus with besides
But I prefer "making her dismiss you" with the whole sentence compared to
>Besides, I don't think dismissing you just to make allowance for me is a good thing